api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3211,6 +3211,15 @@ message Identity {
     (gogoproto.jsontag) = "allowed_resource_ids,omitempty",
     (gogoproto.nullable) = false
   ];
+  // PreviousIdentityExpires is the expiry time of the identity/cert that this
+  // identity/cert was derived from. It is used to determine a session's hard
```

Comment:
Again, I wouldn't use the `hard deadline` verbiage, since the default mode is for disconnect on expired cert to be false (which means it isn't actually a "hard deadline" unless you opt-in to that behavior)

Reply:
I think it's accurate now, it says "It is used to determine a session's hard deadline **in cases where both require_session_mfa and disconnect_expired_cert are enabled**."